### PR TITLE
Blog update for 29 May 2018

### PR DESCRIPTION
**Pull request by anumehaagrawal**
Blog posts for Piston , Compsoc and CAS added

### CODE REVIEW

File: blog_src/_posts/2018-05-28-alterego.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Alterego"
+date: 2018-05-28 23:32:44
+image: '/assets/img/'
+description: 'What if we could just think of the question and a device could just Google it real quick and give the answer?'
+tags:
+- Alterego
+categories:
+- CompSoc
```
Comment (salman-bhai) [inline blog_src/_posts/2018-05-28-alterego.md]: Is this CompSoc 🙈🙈🙈?

File: blog_src/_posts/2018-05-28-why-50hz.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+layout: post
+title: "Why 50Hz ?"
+date: 2018-05-28 23:32:44
+image: '/assets/img/'
+description: 'Why does Power Supply Systems run only at 50 or 60 Hz and not any other frequency?'
+tags:
+- electric power grid
+- 50Hz frequency
+categories:
+- CompSoc
```
Comment (salman-bhai) [inline blog_src/_posts/2018-05-28-why-50hz.md]: This is definitely not CompSoc 😝😜

File: blog_src/_posts/2018-05-28-market-basket.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+Support (Keyboard -> Mouse) =  ![Support](/blog/assets/img/market-basket/support.png)
+
+#### Confidence
+The measure of certainty or trustworthiness associated with each discovered pattern.
+Confidence (Keyboard -> Mouse) = ![Confidence](/blog/assets/img/market-basket/conf.png)
+
+MBA is being used in all aspects of business from small scale retailing to large scale marketing .Its being used extensively in the food industry as well .Suppose that the sales report in a confectionary shop shows that on the first Monday of the month of 1,000 customers, 200 bought cupcakes, and from those who bought cupcakes , 50 bought chocolates as well . Here we see a rule that if someone buys cupcakes, also buys chocolates, which support is 5% (50/1000), and the confidence is 25% (50/200). 
+IEEE as a professional club provides learning opportunities within the engineering sciences, research, and technology. The goal of the IEEE education programs is to ensure the growth of skill and knowledge in the electricity-related technical professions and to foster individual commitment to continuing education among IEEE members, the engineering and scientific communities, and the general public.
```
Comment (salman-bhai) [inline blog_src/_posts/2018-05-28-market-basket.md]: Why is this line needed in this?

**Comment by anumehaagrawal** [comment]
@salman-bhai I have made the necessary changes
